query-engine/query-engine/src/exec_loader.rs

```diff
@@ -90,6 +93,33 @@ async fn mysql(
     Ok((db_name, sql_executor("mysql", mysql, false)))
 }
 
+#[cfg(feature = "sql")]
+async fn mssql(
+    source: &(dyn Source + Send + Sync),
+) -> PrismaResult<(String, Box<dyn QueryExecutor + Send + Sync + 'static>)> {
+    trace!("Loading SQL Server connector...");
+
+    let mssql = Mssql::from_source(source).await?;
+
+    let mut splitted = source.url().value.split(";");
+    splitted.next();
+
+    let mut params: HashMap<String, String> = splitted
+        .map(|kv| {
+            let mut splitted = kv.split("=");
+            let key = splitted.next().unwrap();
+            let value = splitted.next().unwrap();
+
+            (key.to_lowercase(), value.to_string())
+        })
+        .collect();
+
+    let db_name = params.remove("database").unwrap_or_else(|| String::from("master"));
```

Comment:
I don't understand the goal in mutating the URL, is it to default to "master"? If so, why here rather than in quaint?

Reply:
Ah, the prisma models want the database name, so we might as well give it here...